vuln/core/README.txt

```diff
@@ -0,0 +1,631 @@
+ref: https://github.com/nodejs/node/pull/5507
+cve: CVE-2016-0797
+overview:
+Fix a defect that can cause memory corruption in certain very rare cases
+relating to the internal `BN_hex2bn()` and `BN_dec2bn()` functions. It is
+believed that Node.js is not invoking the code paths that use these functions so
+practical attacks via Node.js using this defect are _unlikely_ to be possible.
+
+--
+
+vulnerable: 4.x, 5.x
+patched: ^4.3.2, ^5.7.1
+ref: https://nodejs.org/en/blog/vulnerability/openssl-march-2016/
+ref: https://github.com/nodejs/node/pull/5507
+ref: https://ssrg.nicta.com.au/projects/TS/cachebleed
+cve: CVE-2016-0702
+overview:
+Fix a defect that makes the _[CacheBleed
+Attack](https://ssrg.nicta.com.au/projects/TS/cachebleed/)_ possible. This
+defect enables attackers to execute side-channel attacks leading to the
+potential recovery of entire RSA private keys. It only affects the Intel Sandy
+Bridge (and possibly older) microarchitecture when using hyper-threading. Newer
+microarchitectures, including Haswell, are unaffected.
+
+--
+
+vulnerable: 4.x, 5.x
+patched: ^5.6.0, ^4.3.0
+description: mitigate against the Logjam attack
+overview:
+To mitigate against the Logjam attack, TLS clients now reject Diffie-Hellman
+handshakes with parameters shorter than 1024-bits, up from the previous limit of
+768-bits.
+
+--
+
+vulnerable: 5.x, 4.x
+patched: ^5.1.1, ^4.2.3
+cve: CVE-2015-8027
+overview:
+A bug whereby an HTTP socket may no longer have a parser associated with it but
+a pipelined request attempts to trigger a pause or resume on the non-existent
+parser, a potential denial-of-service vulnerability.
+
+--
+
+XXX this doesn't appear to be remote, is it a node vulnerability?
+
+vulnerable: 5.x, 4.x
+patched: ^5.1.1, ^4.2.3
+cve: CVE-2015-6764
+overview:
+Backport fix for CVE-2015-6764, a bug in v8's `JSON.stringify()` that can result
+in out-of-bounds reads for arrays.
+
+--
+
+vulnerable: 5.x, 4.x
+patched: ^5.1.1, ^4.2.3
+cve: CVE-2015-3193
+ref: http://openssl.org/news/secadv/20151203.txt
+description: BN_mod_exp may produce incorrect results on x86_64
+overview:
+An attack may be possible against a Node.js TLS server using DHE key exchange.
+Details are available at <http://openssl.org/news/secadv/20151203.txt>.
+
+--
+
+vulnerable: 5.x, 4.x
+patched: ^5.1.1, ^4.2.3
+cve: CVE-2015-3194
+ref: https://github.com/nodejs/node/pull/4134
+description: Certificate verify crash with missing PSS parameter
+overview:
+A potential denial-of-service vector for Node.js TLS servers using client
+certificate authentication; TLS clients are also impacted. Details are available
+at <http://openssl.org/news/secadv/20151203.txt>.
+
+--
+
+vulnerable: 4.7.1, 6.9.3
+description: no shasum exists to verify downloads
+overview:
+While promoting additional platforms for v4.7.1 and v6.9.3 after the release,
+the tarballs on the release server were overwritten and now have different
+shasums.
+
+--
+
+XXX this was NOT called a security release?
```

Comment:
since this breaks a security guarantee, seems like it would deserve a CVE @nodejs/security